### PR TITLE
Add unit test for fuse_opt_ops

### PR DESCRIPTION
**Pull request by chengduoZH**

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py [hunk 1]
```diff
@@ -367,6 +364,49 @@ def test_seresnext_with_fused_all_reduce(self):
...
+        check_func_1 = partial(
+            self.check_network_convergence, fuse_all_optimizer_ops=False)
+        check_func_2 = partial(
+            self.check_network_convergence, fuse_all_optimizer_ops=True)
+        # self._check_resnet_convergence(
+        #     SE_ResNeXt50Small,
+        #     check_func_1,
+        #     check_func_2,
+        #     use_cuda=True,
+        #     delta2=1e-3)
```
Comment (chengduoZH) [inline python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py:397]: I will fix this test in next PR.

File: python/paddle/fluid/tests/unittests/test_fuse_optimizer_pass.py [hunk 2]
```diff
@@ -111,7 +59,7 @@ def test_simple_fc_with_fuse_op(self):
 
     def test_batchnorm_fc_with_fuse_op(self):
         self._compare_fused_optimizer_ops(fc_with_batchnorm, True)
-        # self._compare_fused_optimizer_ops(fc_with_batchnorm, False)
+        self._compare_fused_optimizer_ops(fc_with_batchnorm, False)
```
Comment (chengduoZH) [inline python/paddle/fluid/tests/unittests/test_fuse_optimizer_pass.py:62]: Open this unit test.

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py [hunk 3]
```diff
@@ -367,6 +364,49 @@ def test_seresnext_with_fused_all_reduce(self):
...
+        self._check_resnet_convergence(
+            SE_ResNeXt50Small,
+            check_func_1,
+            check_func_2,
+            use_cuda=False,
+            iter=2,
+            delta2=1e-3)
```
Comment (chengduoZH) [inline python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py:404]: Add `fuse_all_optimizer_ops ` unit test in `test_parallel_executor_seresnext.py`.

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py [hunk 4]
```diff
@@ -178,6 +179,15 @@ def optimizer(learning_rate=0.01):
     return optimizer
 
 
+def init_data(batch_size=2):
+    np.random.seed(5)
+    img = np.random.random(size=[batch_size] + img_shape).astype(np.float32)
+    label = np.array(
```
Comment (Yancey0623) [inline python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py]: label.astype("int64") ?

Comment (Yancey0623) [reply python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py]: merge the `init_data` function and simple_nets.init_data ?

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py (hunk 1 above)
Comment (Yancey0623) [reply python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py:397]: Add a TODO comment?

File: python/paddle/fluid/tests/unittests/test_fuse_elewise_add_act_pass.py [hunk 5]
```diff
@@ -12,108 +12,23 @@
...
-    loss = fluid.layers.mean(loss)
-    return loss
-
-
-def fc_with_batchnorm(use_feed):
-    if use_feed:
-        img = fluid.layers.data(name='image', shape=[784], dtype='float32')
-        label = fluid.layers.data(name='label', shape=[1], dtype='int64')
-    else:
-        reader = fluid.layers.open_files(
```
Comment (Yancey0623) [inline python/paddle/fluid/tests/unittests/test_fuse_elewise_add_act_pass.py:58]: no need to test `read_file`?

Comment (chengduoZH) [reply python/paddle/fluid/tests/unittests/test_fuse_elewise_add_act_pass.py:58]: `Recordio` is no longer maintained now, so no need to test `read_file`.

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py (hunk 4 above)
Comment (chengduoZH) [reply python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py]: > label.astype("int64") ?

The type is defined in line 187.

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py (hunk 1 above)
Comment (chengduoZH) [reply python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py:397]: Done

File: python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py (hunk 4 above)
Comment (chengduoZH) [reply python/paddle/fluid/tests/unittests/test_parallel_executor_seresnext.py]: > merge the init_data function and simple_nets.init_data ?

Done

**Review by Yancey0623 — DISMISSED** [review]
lgtm

**Review by Yancey0623 — APPROVED** [review]
lgtm
